concent_api/core/models.py

```diff
@@ -684,3 +685,45 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    created_at = IntegerField()
+
+
+class ConcentUseCase(ChoiceEnum):
+    ForcedAcceptance = 'ForcedAcceptance'
+    AdditionalVerification = 'AdditionalVerification'
+    ForcedPayment = 'ForcedPayment'
+
+
+class DepositClaim(Model):
+    subtask = ForeignKey(Subtask, blank=True, null=True)
+    payer_deposit_account = ForeignKey(DepositAccount)
+    payee_ethereum_public_key = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    concent_use_case = CharField(max_length = 32, choices=ConcentUseCase.choices())
+    amount = DecimalField(max_digits=78, decimal_places=0)
+    tx_hash = CharField(max_length=32, blank=True, null=True, unique=True)
+    created_at = IntegerField()
+    updated_at = IntegerField()
+
+
+    def clean(self) -> None:
+        if self.subtask is None and self.concent_use_case != ConcentUseCase.ForcedPayment.name:
+            raise ValidationError({
```

Comment:
please check if this errors are somewhere logged. If not, please add logging using logging.log_string_message

Reply:
No no, we don't log in models.

Reply:
Looks like it's not necessary.